mdm/mw/udidauth.go

```diff
@@ -0,0 +1,86 @@
+package mw
```

Comment:
There's already a few middlewares around the ack/checkin endpoints. For example, `platform/remove` is a middleware that checks wether the device is blacklisted before allowing the request. 

I'd prefer if we didn't have a generic package name like `mw` for something more specific. Maybe `package mdmauth`?

Reply:
I'd be down with that. But if we're going that route then I'd suggest we change the very generic "builtin" package name we have duplicated on every service to be more in line with what it does. `builtinstorage`, `builtinpersistence`, `buildindb` or somesuch.